mk_verificator/scripts/iperf.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+import salt.client as client
+import texttable as tt
+import itertools
+from netaddr import IPNetwork, IPAddress
+
+# ==========CONFIGURATION PART==========#
+# Define network types their CIDR to be tested
+nets = {"public": "192.0.0.0/24",
+        "private": "192.0.1.0/24"}
+
+# Define nodes to be tested (all pairs will be calculated)
+# you can specify node types like this
+# target_nodes=['ctl*','des*','asc*']
+# you can choose some particular nodes (pay attention to format)
+target_nodes = ['des-01.*', 'asc-01.*']
+# you can even skip nodes from the list (not implemented yet)
+# skipped_nodes = ['cpu-035.jpe1.jiocloud.com']
+# you can define regex by your own (perl-type regex, -E option in salt)
+nodes_regex = ''
+# tbd maybe define networks for each node
+# groups = {'ctl*': ['public'],'asc*': ['public','private']}
+# ==========CONFIGURATION PART==========#
+
+if not nodes_regex:
+    nodes_regex += '('
+    for node in target_nodes:
```

Comment:
just little clarification here - use '{0}' instead of '{}' for compatibility with python 2.6